dcos-metrics.go

```diff
@@ -58,7 +57,12 @@ func main() {
 	if producerIsConfigured("http", cfg) {
 		log.Info("HTTP producer enabled")
 		cfg.Producers.HTTPProducerConfig.DCOSRole = cfg.DCOSRole
-		cfg.Producers.HTTPProducerConfig.CacheExpiry = time.Duration(cfg.Collector.MesosAgent.PollPeriod) * time.Minute * 2
+
+		minCacheExpiry := cfg.Collector.MesosAgent.PollPeriod * 2
+		if cfg.Producers.HTTPProducerConfig.CacheExpiry < minCacheExpiry {
```

Comment:
I'm not sure I really understand this. If the bug was that the cache expiry was set too large, then why are we checking it against a min value?